docs/source/conf.py

```diff
@@ -116,6 +116,9 @@
 .. _UCSC Sequence and Annotation Data: http://hgdownload.cse.ucsc.edu/
 .. _Tute's documentation: https://docs.google.com/document/d/1_Kryc4qAqw1NRezaqDJ1tXUSCbxEkKK4SSi_kZuyHtU/pub
 
+.. _Illumina Platinum Genomes project data: http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_public_data/platinum_genomes.html
```

Comment:
There is another way to refer to pages within the site.  See the following restructured text at the top of http://googlegenomics.readthedocs.org/en/latest/workshops/bioc-2015.html

_We will work with both the :doc:`/use_cases/discover_public_data/1000_genomes` reads and variants and also the :doc:`/use_cases/discover_public_data/platinum_genomes` gVCF variants._